lang/java/reef-tests/src/test/java/org/apache/reef/tests/files/FileResourceTest.java

```diff
@@ -105,8 +107,10 @@ private static Configuration getTestDriverConfiguration(final Set<File> theFiles
    * @throws IOException
    */
   private Set<File> getTempFiles(final int n) throws IOException, InjectionException {
+    final JavaConfigurationBuilder jcb = Tang.Factory.getTang().newConfigurationBuilder();
+    jcb.bindNamedParameter(WorkingDirectoryRootFolder.class, "./target/reef/temp");
     final TempFileCreator tempFileCreator = Tang.Factory.getTang()
-        .newInjector(testEnvironment.getRuntimeConfiguration())
+        .newInjector(jcb.build(), testEnvironment.getRuntimeConfiguration())
```

Comment:
Wouldn't it be better to bind this folder in the `TestEnvironment` instead? That way, *all* tests would use it.